pkg/util/ip/ip.go

```diff
@@ -179,3 +179,18 @@ func MustParseCIDR(cidr string) *net.IPNet {
 	}
 	return ipNet
 }
+
+func CheckIPv6(ip net.IP) error {
+	if ip.To16() == nil || ip.To4() != nil {
+		return fmt.Errorf("invalid IPv6 address: %s", ip.String())
+	}
+	return nil
+}
+
+func MustIPv6(s string) net.IP {
+	ip := net.ParseIP(s)
+	if err := CheckIPv6(ip); err != nil {
+		return nil
```

Comment:
AFAIK, a `MustDoSomething` function just "doSomething" and panics on error, which is used when the input must be valid, for example when it's used in testing or code with static input. 
https://github.com/kubernetes/kubernetes/blob/f840bd719dcfcc42ea2eb5fcdb025ea0dc0b9dfe/staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go#L169
https://github.com/kubernetes/kubernetes/blob/f840bd719dcfcc42ea2eb5fcdb025ea0dc0b9dfe/pkg/kubelet/cm/cpuset/cpuset.go#L279

Can you make the function same style?
```
import utilnet "k8s.io/utils/net"

func MustParseIPv6(s string) net.IP {
	ip := net.ParseIP(s)
	if !utilnet.IsIPv6(ip) {
		panic(fmt.Errorf("invalid IPv6 address: %s", s))
	}
	return ip
}
```

Reply:
Done